internal/services/formatters/service.go

```diff
@@ -187,21 +183,10 @@ func (s *Service) truncatedCode(code string, column int) string {
 }
 
 func (s *Service) GetFilepathFromFilename(filename, projectSubPath string) string {
-	basePath := file.ReplacePathSeparator(path.Join(s.GetConfigProjectPath(), projectSubPath))
-	filepath := file.GetPathFromFilename(filename, basePath)
-	if filepath != "" {
-		return path.Join(projectSubPath, filepath[1:])
-	}
-
-	return path.Join(projectSubPath, filepath)
-}
-
-func (s *Service) GetProjectPathWithWorkdir(projectSubPath string) string {
-	if projectSubPath != "" && projectSubPath[0:1] == string(os.PathSeparator) {
-		return fmt.Sprintf("%s%s", s.GetConfigProjectPath(), projectSubPath)
-	}
+	basePath := filepath.Join(s.GetConfigProjectPath(), projectSubPath)
+	filepathWithFileName := file.GetPathFromFilename(filename, basePath)
 
-	return fmt.Sprintf("%s%s%s", s.GetConfigProjectPath(), string(os.PathSeparator), projectSubPath)
+	return path.Join(projectSubPath, filepathWithFileName)
```

Comment:
Change to `filepath.Join` instead. The [path](https://pkg.go.dev/path) package does not deal with Windows paths.